go/client/logclient_test.go

```diff
@@ -214,3 +223,34 @@ func TestAddJSON(t *testing.T) {
 		}
 	}
 }
+
+func TestGetSTHConsistency(t *testing.T) {
+	hs := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		_, err := w.Write([]byte(`{ "consistency": [ "IqlrapPQKtmCY1jCr8+lpCtscRyjjZAA7nyadtFPRFQ=", "ytf6K2GnSRZ3Au+YkivCb7N1DygfKyZmE4aEs9OXl\/8=" ] }`))
+		if err != nil {
+			return
+		}
+	}))
+	defer hs.Close()
+
+	c := New(hs.URL, &http.Client{})
+
+	tests := []struct {
+		first  uint64
+		second uint64
+		proof  [][]byte
+	}{
+		{1, 3, [][]byte{b64("IqlrapPQKtmCY1jCr8+lpCtscRyjjZAA7nyadtFPRFQ="),
+			b64("ytf6K2GnSRZ3Au+YkivCb7N1DygfKyZmE4aEs9OXl/8=")}},
+	}
+
+	for _, tc := range tests {
+		proof, err := c.GetSTHConsistency(tc.first, tc.second)
+		if err != nil {
+			t.Fatalf("Failed to get consistency proof: %v", err)
+		}
+		if !reflect.DeepEqual(proof, tc.proof) {
+			t.Errorf("GetSTHConsistency(): got\n%v, want\n%v", proof, tc.proof)
```

Comment:
I was advised in one of my reviews to reproduce the input in the error message to make the test failure messages more useful when writing tabulated tests.

t.Errorf("GetSTHConsistency(%d, %d): ..... ", tc.first, tc.second, ....)